src/test_application/DebugObject.cpp

```diff
@@ -194,6 +198,35 @@ void DebugCameraController::update_physics_pre()
     }
 }
 
+MachineUserControl* find_user_control(ActiveScene& rScene, ActiveEnt vehicle)
+{
+    machine_id_t const id = osp::mach_id<MachineUserControl>();
+    // Search all parts
+    std::vector<ActiveEnt> const& parts = rScene.reg_get<ACompVehicle>(vehicle).m_parts;
+    for (ActiveEnt partEnt : parts)
```

Comment:
you should be able to do

for(ActiveEnt partEnt : rScene.reg_get<ACompVehicle>(vehicle).m_parts)